package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "metamask-forwarder",
```

Comment:
Let's publish this under `@MetaMask` on GitHub, we can update this field and I can publish it once this is merged

Reply:
We can also deprecate this existing package

Reply:
No need - it was never published